traitcuration/static/js/table-filtering.js

```diff
@@ -0,0 +1,33 @@
+var options = {
+  valueNames: ["name", "curie", "label", "status"],
+};
+
+var table = new List("traits-table", options);
+
+function search(searchstring) {
+  table.search(searchstring, ["name", "curie", "label"]);
+}
+
+function filterClicked(status, activeClass) {
```

Comment:
Could you add some documentation to this script? I'm thinking one top-level description (like a docstring) + some comments throughout the code, at least for the major branching points. They don't have to be verbose